### PR TITLE
TensorView cleanup

### LINKED ISSUE #641: Cleanup TensorView
After the lowering refactor we should remove remaining usage and implementation of the following:
`TensorView::getComputeAtPos`
`TensorView::getComputeAtAxis`.
`relative_compute_at_axis_`
`getRelativeComputeAtAxis`
There are few remaining uses and they should be removed as this is no longer how we're going to view computeAt. This work could probably go with the coming up compute at refactor.

### PR DESCRIPTION
**Pull request by naoyam**
Remove `TensorView::relative_ca_pos_` and `TensorView::compute_at_view_`. These are not necessary anymore since we construct and use the three `ComputeAtMap` maps instead.

Closes #641

### CODE REVIEW

File: torch/csrc/jit/codegen/cuda/ir_interface_nodes.h [hunk 1]
```diff
@@ -320,11 +305,7 @@ class TORCH_CUDA_CU_API TensorView : public Val {
     domain_ = td;
   }
 
-  // Set all computeAt members without checking any correctness. Useful for
-  // computeAt with outputs relative to eachother
-  void setComputeAt(TensorView* computeAtView, int thisPos, int relPos);
-
-  void setComputeAt(int thisPos);
+  void setComputeAt(unsigned int thisPos);
```
Comment (tlemo) [inline torch/csrc/jit/codegen/cuda/ir_interface_nodes.h]: naming: `this_pos`

File: torch/csrc/jit/codegen/cuda/tensor_view.cpp [hunk 2]
```diff
@@ -169,144 +167,16 @@ IterDomain* TensorView::axis(int pos) const {
   return domain()->axis(pos);
 }
 
-void TensorView::setComputeAt(
-    TensorView* computeAtView,
-    int thisPos,
-    int relPos) {
+void TensorView::setComputeAt(unsigned int thisPos) {
```
Comment (tlemo) [inline torch/csrc/jit/codegen/cuda/tensor_view.cpp]: naming...

**Review by tlemo — APPROVED** [review]
One small comment

File: torch/csrc/jit/codegen/cuda/ir_interface_nodes.h (hunk 1 above)
Comment (naoyam) [reply torch/csrc/jit/codegen/cuda/ir_interface_nodes.h]: Thanks. Changed.

**Review by csarofeen — APPROVED** [review]
LGTM

**Pull request merged**